integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -211,9 +211,20 @@ public static void deletePVC(String pvcName, String namespace, String domainUid)
   public static ExecResult exec(String cmd) throws Exception {
     ExecResult result = ExecCommand.exec(cmd);
     if (result.exitValue() != 0) {
-      logger.info("Command " + cmd + " failed with " + result.stderr() + " \n " + result.stdout());
+      logger.info(
+          "Command "
+              + cmd
+              + " failed with stderr = "
+              + result.stderr()
+              + " \n stdout = "
+              + result.stdout());
```

Comment:
I suspect this might be too verbose for the success case.   Before the refactoring, most of the exec calls were silent on success, and I recall the reason is that the test output can become hard to read if the output from all the execs is commingled.    Perhaps (a) default to suppressing, (b) add a method signature with a boolean that sets verboseness for use by those cases that need it, and maybe (c) add an env variable 'VERBOSE_EXEC' or similar (perhaps there's already a general VERBOSE?) that can be used to force verboseness if true...

Reply:
Logging is done only for failure case here. On success case, caller will log the message if needed.

Reply:
Oops, that's a big miss!  Sorry I didn't catch that.  Ok, let's reverse the discussion accordingly:    How will a caller get access to stdout on success so that they can optionally log it if they want?    How will we optionally able to see (via debugging or something) what the stdout/stderr looks like on success for all cases (implies some sort of debug/verbose flag)?

Reply:
The method returns ExecResult with which the stdout and stderr can be accessed. Do we really want to see the debugging on success for all cases? It will be too verbose. For most of the cases, currently stdout/stderr is logged where the debugging is useful on success cases as well. For log comparison purposes, if someone wants to see the debugging on success for a particular case where its currently not logged, then it needs a change to log the message.

Or if you are suggesting to log stdout/stderr on all success cases based on VERBOSE flag, that can be done too..Default is false, so no logging on success at all. If its changed to true, it will be too verbose.